cert-deployer

```diff
@@ -23,3 +23,10 @@ for i in $(env | grep -o KUBECFG_.*=); do
     chmod 600 $SSL_CRTS_DIR/$name
   fi
 done
+
+
+if [ ! -z ${ETCD_UID} ] && [ ! -z ${ETCD_GID} ]; then
+  setfacl -R -m m::rX ${SSL_CRTS_DIR}
```

Comment:
Adding this adds a dependency on ACL which is usually available but we see the weirdest setup, can we check if it succeeds and else log an error? Can we also add a comment describing what we are doing here so people without Linux ACL knowledge know whats happening (or knows what the intention is)

Reply:
I install the `acl` in the container, but not having ACLs enabled in the kernel is probably out of scope in our case. We don't catch any output in this script, so anything will be sent to the container's stdout/stderr.